src/pnpq/devices/waveplate_stub.py

```diff
@@ -0,0 +1,223 @@
+import logging
+import time
+
+from pnpq.errors import (
+    DevicePortNotFoundError,
+    DeviceDisconnectedError,
+    WavePlateMoveNotCompleted,
+    WavePlateHomedNotCompleted,
+    WavePlateGetPosNotCompleted,
+    WavePlateCustomRotateError,
+    WaveplateInvalidStepsError,
+    WaveplateInvalidDegreeError,
+    WaveplateEnableChannelError,
+    WaveplateInvalidMotorChannelError,
+)
+
+class WaveplateStub:
+    """Stub Waveplate Device Class"""
+
+    def __init__(self):
+        # Stub Serial Number
+        # TODO: Custom serial number from initializer
+        self.device_sn = "stubwaveplate"
+
+        # Resolution of the device in steps per degree
+        self.resolution: int = 136533
+        # Maximum steps the device can move (360 degrees)
+        self.max_steps: int = 136533*360
```

Comment:
I'll add a commit here myself. In the real wave plate driver `max_step` seems to equal `resolution`, which means in the check for `step` function the wave plate can only rotate 1 degree. Is this intended? Right now I temporarily multiplied it by 360 degrees for it to pass the tests. 
Let me know if this is the wrong intended behavior. 
@zigen @hhatpsi